packages/interactivity/src/scopes.ts

```diff
@@ -120,23 +104,29 @@ export const getElement = () => {
  *        context.overridableProp = serverContext.overridableProp;
  *      },
  *    },
- *  });
+ *  } );
  * ```
  *
- * @param namespace Store namespace. By default, the namespace where the calling
- *                  function exists is used.
- * @return The server context content.
+ * @param namespace Store namespace. By default, it inherits the namespace of
+ *                  the store where it is defined.
+ * @return The server context content for the given namespace.
  */
-export const getServerContext = < T extends object >(
+export const getServerContext: ( < T extends object >(
 	namespace?: string
-): T => {
+) => DeepReadonly< T > ) & { subscribe?: number } = < T extends object >(
+	namespace?: string
+): DeepReadonly< T > => {
```

Comment:
Same here as in https://github.com/WordPress/gutenberg/pull/72381/files#r2436979218.


```suggestion
) => DeepReadonly< T > ) & { subscribe?: number } = ( namespace ) => {
```

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/72381/commits/de6784875d07608a646033716b76efdbc99a80fa.